scripts/generateDocs.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs')
+const path = require('path')
+
+const README_DIR = './src/stories/README';
+const REPLACE_TARGET = '{% COMPONENT_DOCS %}'
+
+const fileNames = fs.readdirSync('./src/stories/README')
+
+const docs = []
+
+for (const name of fileNames) {
```

Comment:
NIT: How about using a `.map()` like we do right after?

Reply:
yep good idea